src/Compilers/CSharp/Test/Semantic/Semantics/ReadOnlyStructsTests.cs

```diff
@@ -1455,16 +1455,13 @@ public struct S
 ";
             var moduleMetadata = CreateCompilation(csharp, options: TestOptions.DebugModule, targetFramework: TargetFramework.Mscorlib45).EmitToImageReference();
             var moduleComp = CreateCompilation("", new[] { moduleMetadata });
-            var moduleGetterAttributes = moduleComp.GetMember<PropertySymbol>("S.P1").GetMethod.GetAttributes();
```

Comment:
>moduleGetterAttributes [](start = 16, length = 22)

Why remove the validation of emitted attributes?

Reply:
Now the `IsReadOnlyAttribute` is filtered out of the list returned by `GetAttributes()`, so we have to check via symbol APIs (unless I move this test over to use `symbolValidator`, I believe?)

Reply:
It would be good to still check how the attributes are emitted. If that's already covered elsewhere, then it's fine to remove here though.